src/include/pbs_db.h

```diff
@@ -866,6 +531,21 @@ int pg_db_delete_svrattr(pbs_db_conn_t *conn, pbs_db_obj_info_t *obj);
  */
 int compare_obj_hash(void *qs, int len, void *oldhash);
```

Comment:
My bad, but compare_obj_hash() also a generic util API and is not part of libdb actually. It is used by the application, so we should move the declaration to libutil.h and definition to misc_util.c (or similar)